### ISSUE TITLE
Missing a license

### ISSUE THREAD

**Issue by Panquesito7** [minetest/blog#8]
Title says it all.

**Comment by MisterE123** [comment]
I believe this is finished?

**Comment by Panquesito7** [comment]
> I believe this is finished?

Yeah, the license is stated in the README, but a license file would be better, IMO.
What do you think? Thanks. :)
